src/ploigos_step_runner/step_implementers/shared/argocd_generic.py

```diff
@@ -862,3 +862,52 @@ def _argocd_get_app_manifest(
             ) from error
 
         return argocd_app_manifest_file
+
+    @property
+    def kube_api_uri(self):
+        """
+        :return: the k8s API endpoint
+        """
+        return self.get_value('kube-api-uri')
+
+    @property
+    def kube_api_token(self):
+        """
+        :return: true if the kube api token should be persistent
```

Comment:
you changed doc format then we used every where else:

https://github.com/ploigos/ploigos-step-runner/blob/main/src/ploigos_step_runner/step_implementers/deploy/argocd_deploy.py#L178-L181

Reply:
gah. I inherited this code so I didn't even think to check. Let me update for consistency!

Reply:
Documentation updates pushed! Please review and resolve this conversation if the changes are acceptable.